feature-group-definitions/trusted-types.yml

```diff
@@ -0,0 +1,37 @@
+spec: https://w3c.github.io/trusted-types/dist/spec/
+caniuse: trusted-types
+usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/2722
+status:
+  baseline: false
+  support:
+    chrome: "83"
+    chrome_android: "83"
+    edge: "83"
+compat_features:
+  - api.TrustedHTML
+  # - api.TrustedHTML.toJSON
```

Comment:
What is the rationale for excluding the `toJSON` methods?

Support in Chromium and likely the method itself have been added a bit later on, but then that wouldn't change its baseline status in any case (more than 30 months since version 90 was shipped, and feature is not supported in other browser engines).

Reply:
It's just that Trusted Types can be used since Chrome 83. The version number isn't used on MDN, but if caniuse imports web-features, version numbers would matter. So I don't want to use the wrong version number just because it doesn't change the Baseline status.

What we could do is to include the `toJSON` and treat the version numbers as an editorial override, not matching what we'd infer from BCD. At some point we'll need a "part of the feature but not required to use" annotation I think.